stage1/init/common/mount.go

```diff
@@ -80,3 +84,25 @@ func GenerateMounts(ra *schema.RuntimeApp, volumes map[types.ACName]types.Volume
 
 	return ra.Mounts
 }
+
+// PrepareMountpoints creates and sets permissions for volumes.
+func PrepareMountpoints(path string, vol *types.Volume) error {
+	if vol.Kind == "empty" {
+		diag.Printf("creating an empty volume folder for sharing: %q", path)
+		err := os.MkdirAll(path, 0755)
```

Comment:
mount.go and pod.go are in the same package `common`. You should be able to use `sharedVolPerm` instead of 0755.